File: src/lib/ipc/logs.ts
import { ipcInvoke } from './_invoke';

export async function listAppLogs(limit = 500): Promise<string[]> {
  return ipcInvoke<string[]>('list_app_logs', { limit });
}

export async function clearAppLogs(): Promise<void> {
  await ipcInvoke('clear_app_logs');
}

File: core/src-tauri/src/commands/logs.rs
use crate::app_logs;

#[tauri::command]
pub fn list_app_logs(limit: Option<usize>) -> Vec<String> {
    app_logs::list_logs(limit.unwrap_or(500))
}

#[tauri::command]
pub fn clear_app_logs() {
    app_logs::clear_logs();
}
